pubspec.yaml

```diff
@@ -94,7 +94,7 @@ dependencies:
   rich_text_controller: ^1.4.2
   rxdart: ^0.28.0
   scroll_to_index: ^3.0.1
-  searchfield: ^1.1.6
+  searchfield: ^1.2.4
```

Comment:
`master` branch is pinned at Flutter 3.24.5 so don't change this. You can use [fvm](https://fvm.app/) to have multiple Flutter SDK versions